src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -201,7 +203,7 @@ export async function makePythonDebugConfig(
         type: 'python',
         request: 'attach',
         runtimeFamily: RuntimeFamily.Python,
-        outFilePath: outFilePath,
+        outFilePath: pathutil.normalize(outFilePath ?? ''),
```

Comment:
In what cases will `outFilePath` be undefined? Should that be an error case? What happens if we do end up running this with `pathutil.normalize('')` ?

Reply:
>  what cases will `outFilePath` be undefined?

that kind of refinement will be worked out iteratively as we refine the "config" phase.

> What happens if we do end up running this with `pathutil.normalize('')` ?

it returns empty string. covered by tests.